### ISSUE TITLE
[Bug] Tabby ex command not found

### ISSUE THREAD

**Issue by Ajaymamtora** [nanozuki/tabby.nvim#148]
I'm trying to set a tab name but it says command not found

```lua
   Error  21:13:11 msg_show.emsg

E492: Not an editor command: Tabby 

```

**Comment by nanozuki** [comment]
What's your config for tabby? I can not reproduce this bug.

**Comment by nanozuki** [comment]
Oh... The question is, `tab name` is available from version `2.0`. So it can't work in `1.*` version config.

When I tried to provide a new function 'tabname', I realized it's difficult to add this to the design of version `1.*`, so I provided a new version for tabby.nvim.
